src/main/java/ti4/service/agenda/LookAgendaService.java

```diff
@@ -22,6 +22,9 @@ public static void lookAtAgendas(Game game, Player player, int count, boolean lo
 
     private static List<MessageEmbed> getAgendaEmbeds(int count, boolean fromBottom, Game game) {
         List<MessageEmbed> agendaEmbeds = new ArrayList<>();
+        if (count > game.getAgendas().size()) {
+            count = game.getAgendas().size();
+        }
```

Comment:
Help with a few random attempts to look at all agendas.
A few examples in a row: https://discord.com/channels/943410040369479690/1341438476528123904/1376470594593357948